cfme/tests/infrastructure/test_config_management_rest.py

```diff
@@ -230,3 +231,74 @@ def test_authentications_options(self, appliance, config_manager):
         collection = appliance.rest_api.collections.authentications
         assert 'credential_types' in collection.options()['data']
         assert_response(appliance)
+
+
+@pytest.fixture(scope="function")
+def config_manager_rest(config_manager_obj):
+    """Creates provider using REST API."""
+    config_manager_obj.create_rest()
+    assert_response(config_manager_obj.appliance)
+    rest_entity = config_manager_obj.rest_api_entity
+    rest_entity.reload()
+
+    yield rest_entity
+
+    if rest_entity.exists:
+        rest_entity.action.delete()
+        rest_entity.wait_not_exists()
+
+
+@pytest.mark.tier(3)
+def test_config_manager_create(config_manager_rest):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Rest
+        caseimportance: medium
+        initialEstimate: 1/10h
+        testSteps:
+            1. Add Ansible Tower via REST
+        expectedResults:
+            1. Provider must be added and validated successfully.
+
+    Bugzilla:
+        1621888
+    """
+    assert "?provider_class=provider" in config_manager_rest.href
+    assert "::Provider" in config_manager_rest.type
+
+
+def test_config_manager_edit(request, config_manager_rest):
+    """Test editing a config manager using REST API.
+
+    Metadata:
+        test_flag: rest
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Rest
+        caseimportance: high
```

Comment:
Another observation , but otherwise the PR looks good.

The test_config_manager_edit test has been marked to be of high importance , whereas the remaining tests are marked to be of medium importance.